src/server/game/AI/SmartScripts/SmartScriptMgr.cpp

```diff
@@ -975,10 +975,13 @@ bool SmartAIMgr::IsEventValid(SmartScriptHolder& e)
                 return false;
             break;
         case SMART_ACTION_PLAY_ANIMKIT:
-            if (e.action.animKit.animKit && !IsAnimKitValid(e, e.action.animKit.animKit))
-                return false;
+			if (e.action.animKit.type != 4)
```

Comment:
This doesn't make sense, IsAnimKitValid should be in charge of that, then you just log if > 4

Reply:
done.
removed type 4 and created separate action for that, because it was not very good to combine 
PLAY_ANIM_KIT and PLAY_SPELL_VISUAL together.